### ISSUE TITLE
Modals should have a close button

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#354]
It is recommend the show/hide column dialog on tables has a visible close button. Ensure it is keyboard accessible - able to reach via tab/shift+tab through the app, and enter to close. Ensure it has a label that says close.
